### DISCUSSION TITLE
Improve and simplify Dependabot configuration

### DISCUSSION THREAD

**Discussion by rfc-st** [community/community#14123] [Code Security] [answered]
I have a humble and small repository, and for which I simply want to configure its Dependabot. Having to read, understand and configure the options present in https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates seems to me a real waste of time.

Please: simplify this process as much as possible or show a wizard to configure it. Otherwise it loses its usefulness.

Thanks.

**Comment by exvuma** [comment, marked as answer]
Thanks for the feedback! We are working to make enablement for version updates easier. For now, you can go to Insights > Dependency Graph > Dependabot on your repo and see an option to auto-create the file. Also, you can enter `:you-repo-url//new/main?dependabot_template=1&filename=.github%2Fdependabot.yml`. Does this help in the meantime? Thanks for your input and your patience while we work on making this more seamless in the UI.
